Gemfile

```diff
@@ -0,0 +1,46 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
```

Comment:
What does this line mean/do?

Reply:
detail: https://stackoverflow.com/questions/41454333/meaning-of-new-block-git-sourcegithub-in-gemfile